src/requireAccess.ts

```diff
@@ -1,61 +0,0 @@
-import * as _ from "lodash";
-import * as clc from "cli-color";
-
-import * as getInstanceId from "./getInstanceId";
-import * as getProjectId from "./getProjectId";
-import { FirebaseError } from "./error";
-import * as identifierToProjectId from "./identifierToProjectId";
-import * as requireAuth from "./requireAuth";
-
-export function requireAccess(options: any): Promise<any> {
```

Comment:
What did this ever even do:
```
enableApis = requireAccess({ project: projectId }, [scopes.CLOUD_PLATFORM])
```

Looks like `requireAccess` took a single `options` argument but we were giving it two args?

Reply:
`¯\_(ツ)_/¯`

Reply:
Wanna maybe turn on debug logging and see if that scope is actually sent anywhere at any point?

Reply:
`requireAuth` does have its own scopes that it sets and checks, which is called through `requirePermissions` as well. No logic is being missed by deprecating this path for the init paths.

The only odd thing is `getInstanceId` which would have set `options.instance` which is the RTDB instance, but I don't see that being used in either the functions or firestore codepaths either.

Reply:
Should we add a call to `requireInstance` in places that called `requireAccess` to match the previous functionality.  `options.instance` is only set in two places (`requireAccess` and `requireInstance`) but it's used many places in the codebase.

Reply:
No. The options objects that we were passing into `requireAccess` were the things having `instance` being added onto them, and those objects were never used again. There's no reason to make the call to `requireInstance` since that information never made it back to the callees anyway.

Reply:
Ohhh right.  Man ... `any`.